test/rubygems/test_gem_request.rb

```diff
@@ -338,20 +338,6 @@ def test_user_agent_revision
     util_restore_version
   end
 
-  def test_user_agent_revision_missing
```

Comment:
I removed this test because there is no platform and versions that didn'T have `RUBY_REVISION` today.